PowerToys.sln

```diff
@@ -1,602 +1,616 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Can you please fix the git history changes for solution file to only include necessary changes?